tests/fundAccountPage.test.ts

```diff
@@ -0,0 +1,148 @@
+import { test, expect } from "@playwright/test";
+
+test.describe("Fund Account Page", () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto("http://localhost:3000/account/fund");
+  });
+
+  test("Shows testnet network in the title by default", async ({ page }) => {
+    await expect(page.getByTestId("networkSelector-button")).toHaveText(
+      "Testnet",
+    );
+    await expect(page.locator("h1")).toHaveText(
+      "Friendbot: fund a testnet network account",
+    );
+  });
+
+  test("Shows futurenet network in the title if I change my network to futurenet", async ({
+    page,
+  }) => {
+    // Click network selector dropdown button
+    await page.getByTestId("networkSelector-button").click();
+    await expect(page.getByTestId("networkSelector-dropdown")).toBeVisible();
+
+    // Select Futurenet in the dropdown list
+    await page
+      .getByTestId("networkSelector-dropdown")
+      .getByTestId("networkSelector-option")
+      .filter({ has: page.getByText("Futurenet") })
+      .click();
+
+    // Network Submit button
+    const submitButton = page.getByTestId("networkSelector-submit-button");
+
+    // Select 'Futurenet' in the network dropdown list
+    await expect(submitButton).toHaveText("Switch to Futurenet");
+    await expect(submitButton).toBeEnabled();
+
+    // Click 'Switch to Futurenet' button
+    await submitButton.click();
+
+    await expect(page.getByTestId("networkSelector-button")).toHaveText(
+      "Futurenet",
+    );
```

Comment:
We could simplify the network selection by setting a URL. We test Network selection separately, so we can assume that it's working here.

```suggestion
    await page.goto("http://localhost:3000/account/muxed-create?||=network$id=futurenet&label=Futurenet&horizonUrl=https:////horizon-futurenet.stellar.org&rpcUrl=https:////rpc-futurenet.stellar.org&passphrase=Test%20SDF%20Future%20Network%20/;%20October%202022;;")
`